plugins/catalog-backend/src/ingestion/processors/UrlReaderProcessor.test.ts

```diff
@@ -57,7 +62,12 @@ describe('UrlReaderProcessor', () => {
 
   it('should fail load from url with error', async () => {
     const logger = getVoidLogger();
-    const reader = UrlReaders.default({ logger, config: new ConfigReader({}) });
+    const reader = UrlReaders.default({
+      logger,
+      config: new ConfigReader({
+        backend: { reading: { allow: [{ host: 'localhost' }] } },
```

Comment:
For local testing, is there any need to add explicitly (or in the default app-config) typical localhost addresses / IPs? Obv not `*.local`, but `localhost` et al

Reply:
Not a very common thing to do I think?

Reply:
Not sure!